### PR TITLE
fix: Fixes bug in readme contributor badge

### PR DESCRIPTION
**Pull request by abhpd**
image
image

### CODE REVIEW

**Comment by ramsy0dev** [comment]
can you change the 
``` 
(#contributors-)
```
into 
``` 
(#contributors)
```
because if you click that badge it will not scroll the contributors section while there is that ```-``` in there.

File: README.md [hunk 1]
```diff
@@ -1,18 +1,20 @@
-<p align="center">
+<div align="center">
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+
 [![All Contributors](https://img.shields.io/badge/all_contributors-6-orange.svg?style=flat-square)](#contributors-)
```
Comment (PickleNik) [inline README.md]: needs `#contributors` instead of `#contributors-`, otherwise the link doesn't work

**Review by PickleNik — CHANGES_REQUESTED** [review]
Screen Shot 2022-08-20 at 5 05 41 PM
This doesn't look like the screenshot you've sent.

Please take a look at #119 for centering the badge and change the link as suggested above.
Thank you :)

**Comment by abhpd** [comment]
> Screen Shot 2022-08-20 at 5 05 41 PM
> 
> This doesn't look like the screenshot you've sent.
> Please take a look at #119 for centering the badge and change the link as suggested above. Thank you :)

@PickleNik I think there is some issue with the github preview. You can check my fork here: https://github.com/abhpd/exer_log/tree/readme-contributor-badge-fix

**Comment by abhpd** [comment]
I think #119 is also a valid solution. If we're satisfied with it we can close this PR.

**Comment by PickleNik** [comment]
> @PickleNik I think there is some issue with the github preview. You can check my fork here: https://github.com/abhpd/exer_log/tree/readme-contributor-badge-fix

You're right, the preview is weird.

**Comment by PickleNik** [comment]
> I think #119 is also a valid solution. If we're satisfied with it we can close this PR.

Your solution is more slim, I like it, however the [bot rc badgeTemplate](https://github.com/KalleHallden/exer_log/blob/alpha/.all-contributorsrc#L65) needs to be changed to match. (every time Kalle uses the bot to add contributors to the readme, the bot modifies the section between the `<!-- ALL-CONTRIBUTORS-BADGE:START -->` and `<!-- ALL-CONTRIBUTORS-BADGE:END-->` comments with the badgeTemplate).

(edit: seems like #129 is taking care of that already)

**Comment by PickleNik** [comment]
@abhpd, there is 3 PRs for this little readme bug 😅, I think #129 covers it best, thanks for all the effort, but I think it's best to close this in favor of #129 to reduce the load on Kalle in checking all of them :)

**Comment by abhpd** [comment]
@PickleNik Sure thanks 🙌

**Pull request closed without merging**